tests/countries/test_turks_and_caicos_islands.py

```
@@ -0,0 +1,156 @@
+
+    def test_commonwealth_day(self):
+        self.assertHolidayName("Commonwealth Day", (f"{year}-03-{day}" for year, day in 
+                             [(2020, 9), (2021, 8), (2022, 14), (2023, 13), (2024, 11), (2025, 10)]))
+                             
+    def test_good_friday(self):
+        self.assertHolidayName("Good Friday", (f"{year}-{month}-{day}" for year, month, day in
+                             [(2020, 4, 10), (2021, 4, 2), (2022, 4, 15), (2023, 4, 7), (2024, 3, 29), (2025, 4, 18)]))
+                             
+    def test_easter_monday(self):
+        self.assertHolidayName("Easter Monday", (f"{year}-{month}-{day}" for year, month, day in
+                             [(2020, 4, 13), (2021, 4, 5), (2022, 4, 18), (2023, 4, 10), (2024, 4, 1), (2025, 4, 21)]))
```

Ruff (0.8.2)

Line too long (119 > 99)

(E501)